source/LoadInsight.py

```diff
@@ -0,0 +1,109 @@
+import pandas as pd
+
+def main() :
+	"""Dummy main to test basic pipeline functionality"""
+	input1 = DataObject()
+	output1 = DataObject()
+	transform = Transformation(inputs=[input1],outputs=[output1],function=nop)
+	p1 = Pipeline()
+	p1.add(transform)
+	p1.run()
+
+def nop(x):
+	return(x)
```

Comment:
I'm not a fan of `noop` functions like this because future cleanup can we be painful. Nonetheless, this is minor but I would leverage a `pass` here instead of a return. Better yet, we can remove the function and simply inline `lambda *args: None`.